google/datalab/utils/commands/_utils.py

```diff
@@ -461,7 +461,7 @@ def parse_control_options(controls, variable_defaults=None):
   div_id = _html.Html.next_id()
   if variable_defaults is None:
     variable_defaults = {}
-  for varname, control in list(controls.items()):
```

Comment:
Interesting, I didn't know about this. It looks like it's not required but I'll keep it for safety.